quinn-udp/src/unix.rs

```diff
@@ -349,7 +363,14 @@ unsafe fn sendmmsg_fallback(
     if n == -1 {
         -1
     } else {
-        (*msgvec).msg_len = n as libc::c_uint;
+        #[cfg(not(target_os = "freebsd"))]
+        {
+            (*msgvec).msg_len = n as libc::c_uint;
+        }
+        #[cfg(target_os = "freebsd")]
+        {
+            (*msgvec).msg_len = n as libc::ssize_t;
+        }
```

Comment:
Could avoid this duplication by doing `as _` with a comment to note that the type varies, maybe?

Reply:
Yes. Fixed: [df71882](https://github.com/quinn-rs/quinn/pull/1555/commits/df71882463736b227488c122b01114be7cbdc7c7)